UnitsNet/GeneratedCode/Quantities/VolumeFlow.NetFramework.g.cs

```diff
@@ -1337,6 +1415,36 @@ private double AsBaseNumericType(VolumeFlowUnit unit)
             }
         }
 
+        /// <summary>
+        /// Gets the unit for this quantity compatible with the given <see cref="UnitSystem"/>.
+        /// </summary>
+        /// <param name="unitSystem">The <see cref="UnitSystem"/> to get the compatible units for.</param>
+        /// <returns>The unit for this quantity compatible with the given  <see cref="UnitSystem"/></returns>
+        public static VolumeFlowUnit GetUnitFor(UnitSystem unitSystem)
```

Comment:
Same here, I think we should have keep only one of the two `GetUnitFor` methods. I prefer `UnitSystem` over `BaseUnits` here as well, but at least it should be consistent.